packages/@okta/vuepress-site/guides/sign-into-your-web-app/require-authentication/net/reqautheverything.md

```diff
@@ -1 +1,24 @@
-code example
\ No newline at end of file
+To require authentication for all actions you can add the `Authorize` attribute in the filter config to apply it to every controller.
+
+```
+public class FilterConfig
+{
+    public static void RegisterGlobalFilters(GlobalFilterCollection filters)
+    {
+        ...
+        filters.Add(new AuthorizeAttribute());
+    }
+}
+```
+
+Also, you can opt to create a `BaseProtectedController`and make all your controllers to inherits it:
```

Comment:
`and make all your controllers inherit it:`

Reply:
done